src/api.js

```diff
@@ -2394,13 +2395,17 @@ api.declare({
 
   const result = {
     workers: workers.entries.map(worker => {
-      return {
-        workerGroup: worker.workerGroup,
-        workerId: worker.workerId,
-        firstClaim: worker.firstClaim.toJSON(),
-        latestTask: worker.recentTasks.pop(),
-        disabled: worker.disabled,
-      };
+      return Object.assign(
+        {
+          workerGroup: worker.workerGroup,
+          workerId: worker.workerId,
+          firstClaim: worker.firstClaim.toJSON(),
+          latestTask: worker.recentTasks.pop(),
+        },
+        worker.quarantineUntil.getTime() > new Date().getTime() ?
```

Comment:
Are you modeling `quarantineUntil` as an optional property?

Is that wise? compared to either setting it `null` or perhaps even better setting to `'1970-01-01T00:00:00.000Z'`.
In fact if it was quarantined in the past is there anything wrong with having that datetime in there?

Reply:
My reasoning is that we are treating a quarantineUntil date in the past as meaning "not quarantined".  I don't think we should require the caller to do that datetime comparison since the most common reason to look at this field will be to answer "is it quarantined right now".  I wouldn't mind setting it to null instead of omitting the field.

Reply:
hmm, fair enough... I buy this argument.

datetime comparisons are hard :)